test/Microsoft.AspNetCore.Razor.Language.Test/TagHelperBinderSyntaxTreePassTest.cs

```diff
@@ -14,6 +14,132 @@ namespace Microsoft.AspNetCore.Razor.Language
     public class TagHelperBinderSyntaxTreePassTest
     {
         [Fact]
+        public void Execute_CanHandleSingleLengthAddTagHelperDirective()
```

Comment:
Could just have one test for this since they all go through the same path (unless I am missing something). Having three tests feels a bit redundant.